protoc-gen-swagger/main.go

```diff
@@ -0,0 +1,115 @@
+package main
```

Comment:
I have no good example.  But it is reasonable for me to put them together because the two generators must be consistent.

For me, they look to be similar to the relationship between protoc-gen-go and gprc-plugin rather than plugins for different languages.